descriptors/ModuleDescriptor-template.json

```diff
@@ -92,6 +102,11 @@
     }
   ],
   "permissionSets" : [
+    {
+      "permissionName": "circulation-bff.external-users.get",
+      "displayName": "circulation BFF - get externalUser by externalUserId and tenantId",
```

Comment:
There's no such entity as an "external user", also we're returning a collection of them. Let's change this to 
"circulation BFF - find users by externalSystemId in the specified tenant". 
Same in the `description` below.

Reply:
done